src/test/java/DayTwoTest.java

```diff
@@ -2,6 +2,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
+//choice of test data?
```

Comment:
Interesting that you haven't used the test cases from the problem description, was there a reason?